File: Semestru 4/MPP/lab/.net/lab1/lab1/Models/Domain/Signup.cs
namespace lab1.Models.Domain
{
	public class Signup : Entity<Tuple<int, int>>
	{
        public Child Child { get; set; }
        public Event Event { get; set; }

        public Signup(Child child, Event event_)
        {
            this.Child = child;
            this.Event = event_;
		}

        public Signup() { }
    }
}

File: Semestru 4/MPP/lab/java/lab3/Model/src/main/java/org/example/domain/Signup.java
package org.example.domain;

public class Signup extends Entity<Tuple<Integer, Integer>>{
    private Child Child;
    private Event Event;

    public Signup (Child child, Event event) {
        this.Child = child;
        this.Event = event;
    }

    public Signup() {
        this.Child = new Child ("", "");
        this.Event = new Event("", 0, 0);
    };

    public Child GetChild() {
        return this.Child;
    }

    public Event GetEvent() {
        return this.Event;
    }

    @Override
    public String toString() {
        return this.Child.GetName() + " " + this.Event.GetName();
    }
}
